app/javascript/javascripts/admin/index.js

```diff
@@ -0,0 +1,27 @@
+import compareVersion from 'compare-versions';
+const newVesionUrl = 'https://api.github.com/repos/tryzealot/zealot/releases/latest';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/tryzealot/zealot/pullRequest?prid=7817188)
```suggestion
const newVesionUrl = "https://api.github.com/repos/tryzealot/zealot/releases/latest";
```